members/WAL000166.yaml

```diff
@@ -0,0 +1,69 @@
+bioguide: WAL000166
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: "https://app.leg.wa.gov/pbc/memberEmail/49/0"
+    - fill_in:
+        - name: street
+          selector: "#Address_Street"
+          value: $ADDRESS_STREET
+          required: true
+        - name: city
+          selector: "#Address_City"
+          value: $ADDRESS_CITY
+          required: true
+        - name: state
+          selector: "#Address_State"
+          value: $ADDRESS_STATE_POSTAL_ABBREV
+          required: true
+        - name: zipcode
+          selector: "#Address_Zip"
+          value: $ADDRESS_ZIP5
+          required: true
+    - click_on:
+        - value: Validate address
+          selector: "#ValidateAddressButton"
+    - fill_in:
+        - name: firstname
+          selector: "#FirstName"
+          value: $NAME_FIRST
+          required: true
+        - name: lastname
+          selector: "#LastName"
+          value: $NAME_LAST
+          required: true
+        - name: emailaddress
+          selector: "#Email"
+          value: $EMAIL
+          required: true
+        - name: confirmemailaddress
+          selector: "#ConfirmEmail"
+          value: $EMAIL
+          required: true
+        - name: phone
+          selector: "#PhoneNumber"
+          value: $PHONE
+          required: false
+        - name: subject
+          selector: "#CommentSubject"
+          value: $SUBJECT
+          required: true
+        - name: message
+          selector: "#Comment"
+          value: $MESSAGE
+          required: true
+    - find:
+        - selector: input#ResponseRequested_True
+    - click_on:
+        - selector: input#ResponseRequested_True
+    - wait:
+        - value: 2
+    - click_on:
+        - value: Send
+          selector: "#FormSubmitButton"
+  success:
+    headers:
+      status: 200
+    body:
+      contains: The following message was sent to
```

Comment:
You're sure this is the success message for all members?

Reply:
Si. I tested each one individually in the Tester and no failures. All of the success pages look like the one below 

![image](https://user-images.githubusercontent.com/8680733/46759919-8995e000-cc9e-11e8-8a05-3de03cb128e9.png)